src/_guides/language/language-tour.md

```diff
@@ -54,16 +54,30 @@ This content has moved to [Built-in types](/language/built-in-types#booleans).
 
 ### Lists
 
-This content has moved to [Collection types](/language/collection-types#lists).
+This content has moved to [Collections](/language/collections#lists).
+
+<a id="trailing-comma"></a>
```

Comment:
I feel like this change is from another PR...

Reply:
Yep, I based this one on https://github.com/dart-lang/site-www/pull/4647 since I knew that would land first and they affect some of the same content.